tests/migration/online_migration/yast2_migration.pm

```diff
@@ -220,10 +220,20 @@ sub run {
             send_key 'alt-t';
         }
     }
-    assert_screen ['yast2-migration-installupdate', 'yast2-migration-proposal'], 700;
-    if (match_has_tag 'yast2-migration-installupdate') {
+    assert_screen [qw(yast2-migration-installupdate yast2_migration-license-agreement)], 300;
+    if (match_has_tag 'yast2-migration-installupdate', 150) {    # Not all cases have install update message.
         send_key 'alt-y';
+        assert_screen 'yast2_migration-license-agreement', 60;
+        wait_screen_change { send_key "alt-a" };
+        assert_screen 'yast2_migration-license-agreenment-accepted', 60;
+        send_key "alt-n";
     }
+    if (match_has_tag 'yast2_migration-license-agreement', 150) {
+        wait_screen_change { send_key "alt-a" };
+        assert_screen 'yast2_migration-license-agreenment-accepted', 60;
+        send_key "alt-n";
```

Comment:
as you use this same code block twice I suggest to extract that into a function with an explanatory name and call in both places

Reply:
I update the code, and the verification cases are running, I'll update the results later, thanks for your review.